Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/testclasses/AutonomousTest.java

```diff
@@ -3,21 +3,66 @@
 import com.qualcomm.robotcore.eventloop.opmode.Autonomous;
 import com.qualcomm.robotcore.eventloop.opmode.Disabled;
 
-import org.firstinspires.ftc.robotcontroller.external.samples.ConceptCompassCalibration;
 import org.firstinspires.ftc.robotcore.external.Const;
-import org.firstinspires.ftc.robotcore.external.navigation.AngleUnit;
 import org.firstinspires.ftc.robotcore.external.navigation.AxesOrder;
 import org.firstinspires.ftc.robotcore.external.navigation.AxesReference;
-import org.firstinspires.ftc.team6220_PowerPlay.AprilTagDetect;
+import org.firstinspires.ftc.robotcore.external.tfod.Recognition;
+import org.firstinspires.ftc.team6220_PowerPlay.BaseAutonomous;
 import org.firstinspires.ftc.team6220_PowerPlay.Constants;
 
+import java.util.List;
+
 @Disabled
 @Autonomous(name = "AutonomousTest", group = "Test")
-public class AutonomousTest extends AprilTagDetect {
+public class AutonomousTest extends BaseAutonomous {
+    double motorPower;
+    double x;
+    double width = 0;
 
     @Override
     public void runOpMode() throws InterruptedException {
+        // initialize
         initialize();
+
+        // activate TensorFlow on robot camera
+        tfod.activate();
+        tfod.setZoom(1.0, 16.0 / 9.0);
+
+        // grab cone
+        servoGrabber.setPosition(Constants.GRABBER_CLOSE_POSITION);
+
+        // wait for start
         waitForStart();
+
+        // raise slides to stow position
+        driveSlidesAutonomous(Constants.SLIDE_STOW);
+
+        // detect signal
+        detectSignal();
+
+        // drive forward 54 inches
+        // todo - 54?
+        driveInches(0, 54);
+
+        // turn to -45 degrees
+        turnToAngle(-45);
+
+        // todo - 20 pixels?
```

Comment:
Did you do this? This also seems like a magic number that should be a variable.

Reply:
its 45 degrees, but that number is only used there so it doesn't need to be a constant